addons/project/tests/test_project_base.py

```diff
@@ -104,3 +104,30 @@ def test_delete_project_with_tasks(self):
 
         with self.assertRaises(UserError):
             self.project_pigs.unlink()
+
+    def test_auto_assign_stages_when_importing_tasks(self):
+        self.assertFalse(self.project_pigs.type_ids)
+        self.assertEqual(len(self.project_goats.type_ids), 2)
+        first_stage, second_stage = self.project_goats.type_ids
+        self.env['project.task'].create({
+            'name': 'First Task',
+            'user_id': self.user_projectuser.id,
+            'project_id': self.project_pigs.id,
+            'stage_id': first_stage.id
+        })
+        self.assertEqual(self.project_pigs.type_ids, first_stage)
+        self.env['project.task'].create([
```

Comment:
You should test _load_records_create

Reply:
When you write a test, you should check that it fails without your fix. Iinm, it is not the case here.

Reply:
it's already the case, if i comment my fix, the test fail 
![image](https://user-images.githubusercontent.com/64937092/194852617-fdaadcf5-1bea-49c0-bc1a-24e6ad5d0b9b.png)